src/main/java/fm/last/moji/local/LocalMojiFileAttributes.java

```diff
@@ -48,4 +53,17 @@ public String getKey() {
     return mojiFile.getKey();
   }
 
+  @Override
+  public String getChecksum() {
+    InputStream in = null;
+    try {
+      in = mojiFile.getInputStream();
+      return String.format("MD5:%s", Hex.encodeHexString(DigestUtils.md5(in)));
+    } catch (IOException e) {
+      return "ERROR";
```

Comment:
This code isn't actually being called anywhere in this project at the moment so this is all a bit academic but for now why not throw a new RuntimeException(e) here so at least if anything in the future does call this and something goes wrong it won't silently fail. The rest looks fine.